collectors/common.py

```diff
@@ -74,24 +100,30 @@ def fread(file_name=None):
     """
     Simple read function for files of a single value
     :param file_name: (str) file name to read
-    :return: (str) contents of the file
+    :return: (str) contents of the file, or null string for empty file
```

Comment:
"or null string for nonexistent file" (your string doesn't mention nonexistent, and empty files' contents are of course a null string)